### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by chengidesk** [netalertx/NetAlertX#686] [Q&A] [answered]
New to docker because needed NetAlertX, when i follow the instruction i get this error. Wondering what may have gone wrong? 

root@Docker:~# docker run -d --rm --network=host   -v local/path/config:/app/config   -v local/path/db:/app/db   -e TZ=Asia/Kuala_Lumpur   -e PORT=20211   jokobsk/netalertx:latest
docker: Error response from daemon: create local/path/config: "local/path/config" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
See 'docker run --help'.
root@Docker:~# 

Thank you.

**Comment by jokob-sk** [comment, marked as answer]
Hi,

Do you have a folder called `local/path` ? You need to adjust the command to reflect your setup, and replace `local/path` with an existing folder, that will contain 2 folders: `config` and `db`. For example you create a folder this way:

Create the directories:

```sh

mkdir -p /apps/netalertx/config /apps/netalertx/db
```
Adjust the Docker command:
Replace local/path/config and local/path/db with /apps/netalertx/config and /apps/netalertx/db respectively. The adjusted command will look like this:

```sh

docker run -d --rm --network=host -v /apps/netalertx/config:/app/config -v /apps/netalertx/db:/app/db -e TZ=Asia/Kuala_Lumpur -e PORT=20211 jokobsk/netalertx:latest
```
Run the adjusted Docker command:

```sh

docker run -d --rm --network=host -v /apps/netalertx/config:/app/config -v /app
```

**Reply by chengidesk** [reply]
Thank you @jokob-sk for your speedy response ! 
I am really new to docker. I use it because i wanted to explore NetAlertX :)
I'll explore some other discussion by searching first since I see only 2 devices and don't want to waste your valuable time.
Thank you again for your work and your response.

**Reply by jokob-sk** [reply]
You are welcome - happy docker exploring! 🐳😄

TIP: Check your subnets config: https://github.com/jokob-sk/NetAlertX/blob/main/docs/SUBNETS.md

**Reply by chengidesk** [reply]
perfect ! now i have 13 devices found with my home network. 

i'm exploring this solution to be used for my daughter school. their requirement from the gov is each student could only bring a fixed tablet to school. there is no way to manually check that so the only way is to use this NetAlertX and making sure they don't use random MAC. Once an unregistered device show up, the school could then block the mac. 

thank you @jokob-sk for this wonderful system.

**Reply by jokob-sk** [reply]
That's a great use case - hope it fits your needs 👍

**Comment by stanelie** [comment]
Thanks for this post, I ran into the same issue.
